x-pack/plugin/data-frame/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/dataframe/integration/DataFramePivotRestIT.java

```diff
@@ -234,8 +234,8 @@ public void testDateHistogramPivot() throws Exception {
 
         // we expect 21 documents as there shall be 21 days worth of docs
```

Comment:
nit remove or update the comment

Reply:
lulz. Maybe I should just stop writing comments :D